src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -817,14 +989,29 @@ else if ( annotationsToCheckMaf.get(i).equals(MafOutputRendererConstants.FieldNa
             // Don't try to refactor this for-loop to a stream here.
             final List<String> vcfFieldValues = new ArrayList<>();
             for (final VariantContext v: variantContexts) {
+
+                final String transcriptFuncotationName = "Gencode_" + (funcotatorRef.equals("hg19") ? "19" : "28") + "_annotationTranscript";
+
                 final Map<Allele, FuncotationMap> alleleFuncotationMapMap = FuncotatorUtils.createAlleleToFuncotationMapFromFuncotationVcfAttribute(
-                        funcotationKeys, v, "Gencode_19_annotationTranscript", "TEST");
+                        funcotationKeys, v, transcriptFuncotationName, "TEST");
+
                 for (final Allele alternateAllele : v.getAlternateAlleles() ) {
                     final FuncotationMap funcotationMap = alleleFuncotationMapMap.get(alternateAllele);
                     vcfFieldValues.add(funcotationMap.getFieldValue(funcotationMap.getTranscriptList().get(0), annotationToCheckVcf, alternateAllele));
                 }
             }
-            Assert.assertEquals(mafFieldValues, vcfFieldValues, "Failed matching (VCF: " + annotationToCheckVcf + " , MAF: " + annotationToCheckMaf + ")");
+
+            // "Fuzzy" matching between VCF and MAF by allowing blank VCF fields to be Unknown in the MAF.
+            // This is a temporary hold-over until we can do full aliased comparisons of fields.
+            // TODO: Update with better comparison using aliases when they are fully implemented.
+            if ( !Objects.equals(mafFieldValues, vcfFieldValues) ) {
+                Assert.assertEquals(mafFieldValues.size(), vcfFieldValues.size());
+                mafFieldValues.stream().map( v -> {Assert.assertEquals(v, "Unknown"); return true;} );
+                vcfFieldValues.stream().map( v -> {Assert.assertEquals(v, ""); return true;} );
```

Comment:
Isn't this asserting that ALL of the vcf/maf field values must be Unknown/"", in the case where you don't have an exact match across all of the field values? This doesn't seem right...is this code actually getting run?

Reply:
It is being run, but the mapped asserts do not function as expected.

I will fix them.